examples/push/BUILD.bazel

```diff
@@ -15,6 +15,13 @@ oci_push(
     repository = "index.docker.io:9899/<ORG>/image",
 )
 
+oci_push(
+    name = "push_image_repository_file",
+    image = ":image",
+    remote_tags = ["latest"],
+    repository_file = ":repository.txt",
```

Comment:
nit: it would be a more useful example if you stamp this file (use `expand_template` from aspect_bazel_lib)

Reply:
That function isn't in the version of the aspect_bazel_lib (or bazel_skylib) specified in the dependencies, and there's a big warning in oci/dependencies.bzl saying that changing it would be a breaking change, so I'm hesitant to change that.

Reply:
This was just a nit, no need to hold up the merge on it.